### PR TITLE
[IMPROVE] Canned responses

### PR DESCRIPTION
**Pull request by MartinSchoeler**

### CODE REVIEW

**Review by KevLehman — APPROVED** [review]

**Comment by homberger** [comment]
Is this ENTERPRISE only?

File: ee/app/canned-responses/server/settings.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { settings } from '../../../../app/settings';
 
 export const createSettings = () => {
-	settings.addGroup('Canned Responses', function() {
-		this.section('Canned Responses', function() {
+	settings.addGroup('Canned_Responses', function() {
```
Comment (matheusbsilva137) [inline ee/app/canned-responses/server/settings.js:4]: Hey @ggazzo , this is the line that has introduced the duplication.

Comment (KevLehman) [reply ee/app/canned-responses/server/settings.js:4]: Which duplication? 👀

Comment (matheusbsilva137) [reply ee/app/canned-responses/server/settings.js:4]: The one we're fixing in #23056

Comment (KevLehman) [reply ee/app/canned-responses/server/settings.js:4]: Ohhh, noticed it 😬 got it.

**Pull request merged**